src/Dev/Tasks/i18nTextCollectorTask.php

```diff
@@ -6,7 +6,7 @@
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\Debug;
 use SilverStripe\Dev\BuildTask;
-use SilverStripe\i18n\i18nTextCollector;
+use SilverStripe\i18n\TextCollection\i18nTextCollector;
```

Comment:
What do you think about moving text collection to its own module? As I've outlined in my comments around the _t() default value, I don't think text collection is the *only* translation workflow we should support - some teams will just manage their YAML/JSON separately. The reason I'm bringing this up is namespacing: If it's a separate module, we'd probably use `SilverStripe\i18nTextCollection\i18nTextCollector` instead of `SilverStripe\i18n\TextCollection\i18nTextCollector` (since the `SilverStripe\i18n` namespace should be core only)